flows/test/dependent_flows.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python
+
+"""
+Name: Dependent flows testing
+Description: The purpose of this file is to serve as a template to
+    establish a pattern for the implementation of flows moving forward.
+Labels: test
+"""
+
+import os
+import prefect
+import json
+
+# Prefect
+from prefect import Flow, task
+from prefect.tasks.prefect import create_flow_run, get_task_run_result
+from prefect.storage import GitHub
+from prefect.run_configs import UniversalRun
+from prefect.engine.results import PrefectResult
+from prefect.backend import get_key_value
+
+from prefect.tasks.notifications.email_task import EmailTask
+
+# First, we must always define the current environment, and default to staging:
+current_environment = os.getenv("PREFECT_CURRENT_ENVIRONMENT", "staging")
+
+# Notice how test_kv is an object that contains our data as a dictionary:
+environment_variables = get_key_value(key=f"test_kv_{current_environment}")
+
+# Retrieve the email configuration
+email_config = get_key_value(key="aws_email_config")
+
+
+@task(name="First", result=PrefectResult(), slug="first-slug")
+def first():
+    logger = prefect.context.get("logger")
+    logger.info("First Task")
+    return {"users": "thing"}
+
+
+email_task = EmailTask(
+    name="email_task",
+    subject="Test from ATD",
+    email_to=email_config["test_email"],  # <- Type your email here
+    email_from=email_config["email_from"],
+    smtp_server=email_config["smtp_server"],
+    smtp_port=email_config["smtp_port"],
+    smtp_type=email_config["smtp_type"],
+    attachments=None
+)
+
+
+@task(log_stdout=True)
+def format_dict(flow_data):
+    print(f"Got: {flow_data!r}")
```

Comment:
I did a quick search for the `!r` syntax, and I'm wondering, is this a typo, or a future TIL for me.

Reply:
this is a giant typo.